cron/alerts/main.go

```diff
@@ -190,7 +182,7 @@ func getCFRecordsWithSubstring(substring, zoneName string, recs []cloudflare.DNS
 }
 
 func getCFRecords(config AlertsConfig) (map[string][]string, error) {
-	api, err := cloudflare.New(config.CFApiKey, config.CFApiEmail)
+	api, err := cloudflare.NewWithAPIToken(config.CFApiKey)
```

Comment:
Would it be more accurate to call this (and the related env. vars) something like `CFApiToken` now rather than `CFApiKey`?

Reply:
Yes, it would be more accurate. I don't remember why I decided to keep it unchanged.